pandas/tests/io/test_packers.py

```diff
@@ -393,6 +393,12 @@ def categorical_index(self):
         result = self.encode_decode(df)
         tm.assert_frame_equal(result, df)
 
+    def test_timezone_encode_decode(self):
```

Comment:
Can you parameterize using the `tz_aware_fixture`?

Reply:
sure!!